### DISCUSSION TITLE
About usage of sync and create packed pdu block

### DISCUSSION THREAD

**Discussion by ghost** [daniestevez/gr-satellites#461] [Q&A] [answered]
Hello everyone, I have just created a file with binary editor to understand this block and my goal is to match simple preamble. The binary form of the test.bin file is `D2 AB 15 00 12 15 1A CF FC 1D A6 DA AB A0 03 00 00 00 C3 D2 B1 A0 03 00 D2 B1 A0 06 00` and flowgraph is given below. The synchword should match with "1A CF FC 1D" part. However I don't see any output in message debug? I change the repeat parameter in file source to 'Yes', I increase the file size and I remove the throttle block, but still there is no output. 

Can you help me?
![grsatellite](https://github.com/daniestevez/gr-satellites/assets/69161854/1d3eb548-83cf-402e-8dd9-ee1fc9e74945)

**Comment by K4KDR** [comment, marked as answer]
Hello!

Please see the example below for a small change to the flowgraph which will output the 10 bytes from your source file AFTER the SyncWord:

![syncword-test](https://github.com/daniestevez/gr-satellites/assets/19886368/f90849a5-3b86-4369-b220-faf350319f8b)

**Reply by ghost** [reply]
Thank you, I didn't notice that the input of the sync and create packed block must be "bits".
